plugins/duckhunt.py

```diff
@@ -116,7 +116,7 @@ def incrementMsgCounter(event, conn):
     global game_status
     if event.chan in opt_out:
         return
-    if game_status[conn.name][event.chan]['game_on'] == 1 and game_status[conn.name][event.chan]['duck_status'] == 0:
+    if game_status[conn.name][event.chan]['game_on'] == 1 and game_status[conn.name][event.chan]['duck_status'] == 0 and "snoonet/bot/" not in event.host:
```

Comment:
We generally try to avoid network specific code like this, but something like this may work instead:
```python
    # Will return None if the plugin can not be found but doesn't hold a strong reference to it like an import would
    ignore_plugin = event.bot.plugin_manager.find_plugin("ignore")
    if ignore_plugin and ignore_plugin.code.is_ignored(conn.name, event.chan, event.mask.lower()):
        return
```